### PR TITLE
Closes #28: Create a Comment

### LINKED ISSUE #28: Create a Comment
As a commenter, I would like to be able to add a Comment to a Post so that I can take part in the discussion about a Post.

**Given** the user is viewing the Details of a Post
**When** they select the `Add Comment` menu option
**Then** they should be directed to a page with a form for creating a new Comment

**Given** the user has entered the relevant information for a Comment
**When** they click the `Save` button
**Then** the Comment should be saved to the database
**And** the creation datetime should be automatically set to the current date and time
**And** the user should be redirected to the new Comments list page for the related Post  

The "relevant information" for a Comment is

* Subject
* Content

### PR DESCRIPTION
**Pull request by mqmanan**
Closes #28: Create a Comment

Added:
- Comment Module
- Comment Repository with 3 methods (GellAllComments, GetCommentById, AddComment)
- ICommentRepository
- Comment Controller
- AddTransient line in Startup.cs

### CODE REVIEW

File: SQL/02_Tabloid_Seed_Data.sql [hunk 1]
```diff
@@ -17,8 +17,8 @@ values (1, 'C#'), (2, 'JavaScript'), (3, 'Funny'), (4, 'Scary');
 set identity_insert [Tag] off
```
Comment (dalamcd) [inline SQL/02_Tabloid_Seed_Data.sql]: This file should not be included in pull requests

File: Tabloid/Controllers/CommentsController.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+using Tabloid.Models;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```
Comment (dalamcd) [inline Tabloid/Controllers/CommentsController.cs:9]: Need to add [Authorize] to class definition to ensure that users are authenticated and authorized

**Review by dalamcd — CHANGES_REQUESTED** [review]

File: SQL/02_Tabloid_Seed_Data.sql (hunk 1 above)
Comment (mqmanan) [reply SQL/02_Tabloid_Seed_Data.sql]: Deleted file from my branch!

File: Tabloid/Controllers/CommentsController.cs (hunk 2 above)
Comment (mqmanan) [reply Tabloid/Controllers/CommentsController.cs:9]: Added!

**Comment by mqmanan** [comment]
@dalamcd made the changes!

**Review by dalamcd — APPROVED** [review]
Looks great, approved!

**Pull request merged**